templates/Pages/rules_scc_submission.php

```diff
@@ -16,7 +16,15 @@
     <h3>Submission Student Cluster Competition Rules</h3>
 
     <p>
-        Please refer to the IO500 rules webpage; all rules are in effect with three exceptions.
+        Please refer to the 
+    <?php
+                echo $this->Html->link(__('IO500 rules webpage;'), [
```

Comment:
In `templates/Pages/running.php` this text is:

                echo $this->Html->link(__('IO500 Submission Rules'), [

which I think is somewhat better, since "Submission Rules" is the name of that page, and we don't really care that it is a "webpage" or not...

Reply:
sure, seems better

Reply:
Can you please push an update to this branch to replace your original commit, rather than fixing this with a second commit:

    edit templates/Pages/running.php
    git commit -a --amend
    git push -f HEAD:missing_link_scc

Alternately, this PR could be dropped and the `missing_link_scc` branch deleted, and a new patch pushed in its place to the same branch name, but that loses the history of the discussion and is not preferred, and is more work than using `git commit --amend` and `git push -f`.

Reply:
Why not just having a second commit Andreas? I thought that's the preferred way of working?

Reply:
I think having multiple patches, where the first one is bad and a second patch is fixing it is the wrong way to do things. I'd prefer that if there is a problem with a patch, the patch is fixed and pushed again.

Why preserve all of the dirty laundry forever in the history?